Lesson 3 - Objects/code/animals/animals.js

```diff
@@ -1,13 +1,32 @@
 (function (global) {
-	var Cat = null, Bird = null, Worm = null;
+	var Cat = {
+    type:"Cat",
+    eat: function (animal) {
+        if( 
+            ((animal.type =="Bird") && (this.type =="Cat")) || 
```

Comment:
Interesting implementation. Corresponds with the 'spirit' of language :) I will pass it.

Try to implement it using `Object.getPrototypeOf`.